lib/livebook_web/live/hub/teams/app_deployment_list_component.ex

```diff
@@ -0,0 +1,24 @@
+defmodule LivebookWeb.Hub.Teams.AppDeploymentListComponent do
```

Comment:
This component doesn't have state nor events, so it should be a function component instead (can be a private function in the deployment group LV).